Lib/fontbakery/profiles/typenetwork.py

```diff
@@ -1775,7 +1773,7 @@ def com_typenetwork_check_family_duplicated_names(ttFonts):
 )
 
 profile.check_log_override(
-    # From googlefonts.py
+    # From fvar.py
     "com.google.fonts/check/varfont/bold_wght_coord",
```

Comment:
**com.google.fonts/check/varfont/bold_wght_coord** has migrated from `Open Type` profile (fvar.py) to `Google Fonts` profile at https://github.com/fonttools/fontbakery/commit/6f7fcaf2d6389adb5f286ae5c585f8ffe734edb6

Reply:
Thanks I will check this now.

Reply:
Should I manually revert those changes and make a commit on this pull request or just delete the whole PR and make a new one? Probably there is a fancy git method to remove those changes but I’m not sure how.

Reply:
Create an additional commit making the additional changes (reverting the previous ones) then you can push again and I will accept the PR with a squash, which makes all commits into a single one.

Alternatively you could perform the squashing locally. If you would like to learn how to do that, then take a look at the documentation of the `git rebase -i` command.